configuration/components/tracing.libsonnet

```
@@ -79,24 +79,33 @@ function(params) {
   local normalizedName(id) =
     std.strReplace(id, '_', '-'),
 
-  local newCommonLabels(component) =
+  local newCommonLabels(tenant) =
```

Because the non-type system, let's hint it's a string type not a complex JSON for configuration.

```suggestion
  local newCommonLabels(tenantName) =
```